src/test/java/me/damonkelley/tictactoe/PlayersTest.java

```diff
@@ -0,0 +1,17 @@
+package me.damonkelley.tictactoe;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class PlayersTest {
+    @Test
+    public void itGetsAPlayerByMarker() {
+        Player playerA = new Player(Marker.X, null);
+        Player playerB = new Player(Marker.O, null);
+
+        Players players = new Players(playerA, playerB);
+        assertTrue("It can get Player A", playerA == players.get(Marker.X));
```

Comment:
1. Be careful with `==` on instances of `java.lang.Object` - prefer `.equals` pretty much all the time - reference equality is a dangerous habit to get into. On the other hand, it's fine here.
2. `assertTrue` generally gives garbage error messages like "Expected true, got false". Prefer `assertEquals`, or if you reaalllly want reference equality, maybe `assertSame`?